pkgs/lib/libcint/default.nix

```diff
@@ -1,8 +1,8 @@
 { lib, stdenv, fetchFromGitHub, cmake
-, openblas, python
+, blas, python
 } :
 let
-  version = "3.0.15";
```

Comment:
This libcint3, which is(?)/was the  legacy version for pyscf. There is `libcint-4.4.0` is in upstream nixpkgs. I think we can completely remove libcint3 here?